doc/nethogs.8

```diff
@@ -113,7 +117,9 @@ command, as follows:
 sudo setcap "cap_net_admin,cap_net_raw+pe" /usr/local/sbin/nethogs
 .EE
 .in
-
+.SH "Notes"
+1. When using the -P <pid> option, in a case where a process exitted(normal or kernel), Nethogs can not track that it exitted. So, the operating system might create 
+a new process(from another program) with the same pid. In this case, this new process will be showed by Nethogs.
```

Comment:
```suggestion
1. When using the -P <pid> option, in a case where a process exited (normally or abruptly), Nethogs does not track that it exited. So, the operating system might create 
a new process (for another program) with the same pid. In this case, this new process will be shown by Nethogs.
```

Fine to note it here. Adding it to the docs for `-P` above might perhaps be clearer?

Reply:
Yes, for sure!